lib/spree_google_base/feed_builder.rb

```diff
@@ -123,11 +123,11 @@ def build_images(xml, product, variant)
       if Spree::GoogleBase::Config[:enable_additional_images]
         main_image, *more_images = product.master.images
       else
-        main_image = product.master.images.first
+        main_image = product.display_images[:list].first[:portrait_normal_url]
```

Comment:
with the way our data is set up, usually no. but it is possible for it to be nil.

but i think in the scenario in which it's nil `build_images` will exit early and a value for the  `image_link` key won't be set. do you think i should set it to an empty string to be safe?